webpack.config.js

```diff
@@ -1,18 +1,17 @@
 const path = require('path');
-const webpack = require('webpack');
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
 const HtmlWebpackPlugin = require('html-webpack-plugin');
+const MiniCssExtractPlugin = require('mini-css-extract-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+const OptimizeCSSAssetsPlugin = require('optimize-css-assets-webpack-plugin');
+const webpack = require("webpack");
 
 let og_image_url = process.env.RIOT_OG_IMAGE_URL;
 if (!og_image_url) og_image_url = 'https://riot.im/app/themes/riot/img/logos/riot-im-logo-black-text.png';
 
-module.exports = {
+module.exports = (env, argv) => ({
     entry: {
-        // Load babel-polyfill first to avoid issues where some imports (namely react)
-        // are potentially loaded before babel-polyfill.
-        "bundle": ["babel-polyfill", "./src/vector/index.js"],
+        "bundle": "./src/vector/index.js",
```

Comment:
Removing this seems like something I would have expected to be called out, as it has wide potential for subtle breakage. Is this because we're injecting it somewhere else or because we're assuming all browsers we support have the necessary stuff now?

Reply:
Yea, we started targeting the "last 2 versions" of browsers, even before this whole sourcemaps adventure. We can get rid of this polyfill safely, and honestly I forgot about it. It was causing some sourcemap issues too, which just accelerated its exit.

Reply:
Fair enough then